src/ILLink.RoslynAnalyzer/RequiresUnreferencedCodeAnalyzer.cs

```diff
@@ -72,6 +72,12 @@ public override void Initialize (AnalysisContext context)
 						CheckMethodOrCtorCall (operationContext, prop.SetMethod);
 				}, OperationKind.PropertyReference);
 
+				context.RegisterOperationAction (operationContext => {
+					var delegateCreation = (IDelegateCreationOperation) operationContext.Operation;
+					var target = (IMethodReferenceOperation) delegateCreation.Target;
```

Comment:
Same applies here.

Reply:
This diff makes me think we need to share more between the two analyzers. They look like they're doing pretty much exactly the stuff.